suite-native/module-add-coin/src/hooks/useAddCoinAccount.ts

```diff
@@ -0,0 +1,250 @@
+        getAvailableAccountTypesForNetwork({ network })?.[0];
+
+    const getNetwork = useCallback(
+        ({
+            networkSymbol,
+            accountType,
+        }: {
+            networkSymbol: NetworkSymbol;
+            accountType?: AccountType;
+        }) => {
+            const type =
+                accountType ??
+                getDefaultAccountType({
+                    network: supportedNetworks.filter(
+                        network => network.symbol === networkSymbol,
+                    )[0],
+                });
+
+            return supportedNetworks.filter(
+                network =>
+                    network.symbol === networkSymbol &&
+                    (type === 'normal'
+                        ? network.accountType === undefined
+                        : network.accountType === type),
+            )[0];
+        },
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+        [],
+    );
+
+    const showTooManyAccountsAlert = useCallback(() => {
+        showAlert({
+            title: translate('moduleAddCoinAccount.alerts.tooManyAccounts.title'),
+            description: translate('moduleAddCoinAccount.alerts.tooManyAccounts.description'),
+            icon: 'warningCircle',
+            pictogramVariant: 'red',
+            primaryButtonTitle: translate(
+                'moduleAddCoinAccount.alerts.tooManyAccounts.actionPrimary',
+            ),
+            onPressPrimaryButton: () => {
+                hideAlert();
+            },
+        });
+    }, [hideAlert, showAlert, translate]);
+
+    const showAnotherEmptyAccountAlert = useCallback(() => {
+        showAlert({
+            title: translate('moduleAddCoinAccount.alerts.anotherEmptyAccount.title'),
+            description: translate('moduleAddCoinAccount.alerts.anotherEmptyAccount.description'),
+            icon: 'warningCircle',
+            pictogramVariant: 'red',
+            primaryButtonTitle: translate(
+                'moduleAddCoinAccount.alerts.anotherEmptyAccount.actionPrimary',
+            ),
+            onPressPrimaryButton: () => {
+                hideAlert();
+            },
+            secondaryButtonTitle: translate(
+                'moduleAddCoinAccount.alerts.anotherEmptyAccount.actionSecondary',
+            ),
+            onPressSecondaryButton: () => {
+                openLink(
+                    translate('moduleAddCoinAccount.alerts.anotherEmptyAccount.actionSecondaryUrl'),
+                );
+                hideAlert();
+            },
+        });
+    }, [hideAlert, showAlert, translate, openLink]);
+
+    const addCoinAddress = useCallback(
+        ({ network, type }: { network: Network; type?: AccountType }) => {
+            if (!device?.state) {
+                return false;
+            }
+
+            const accountType =
+                type ?? getAvailableAccountTypesForNetwork({ network })?.[0] ?? 'normal';
```

Comment:
please store the `normal` to some const on the top of the file. `const DEFAULT_ACCOUNT_TYPE: AccountType = 'normal;'`.

Reply:
044d93041c